lib/trento/domain/cluster/cluster.ex

```diff
@@ -63,9 +67,12 @@ defmodule Trento.Domain.Cluster do
           hosts: [String.t()],
           selected_checks: [String.t()],
           hosts_executions: [HostExecution.t()],
-          checks_execution: :not_running | :requested | :running
+          checks_execution: :not_running | :requested | :running,
+          rolling_up: boolean()
         }
 
+  def execute(%Cluster{rolling_up: true}, _), do: {:error, :cluster_rolling_up}
```

Comment:
I love pattern matching :heart: so freaking expressive!
For any command on a rolling up cluster go banana! :rocket: